sssom/util.py

```diff
@@ -1505,3 +1507,75 @@ def _get_sssom_schema_object() -> SSSOMSchemaView:
         else SSSOMSchemaView()
     )
     return sssom_sv_object
+
+
+def invert_mappings(
+    df: pd.DataFrame,
+    subject_prefix: str,
```

Comment:
subject_prefix should be optional - it should be possible to flip an entire mapping set without worrying about subject prefix

Reply:
Done